synapse/handlers/identity.py

```diff
@@ -868,6 +870,7 @@ async def ask_id_server_for_third_party_invite(
             "sender": inviter_user_id,
             "sender_display_name": inviter_display_name,
             "sender_avatar_url": inviter_avatar_url,
+            "org.matrix.msc3288.room_type": room_type,  # MSC3288
```

Comment:
Not sure if the extra `MSC3288` comment here adds much?

Reply:
Probably not, habit of consistency due to not all experimental prefixes including the msc number, e.g the room summary one